### DISCUSSION TITLE
Migrate v3 to v4 don't run

### DISCUSSION THREAD

**Discussion by bussekas** [LycheeOrg/Lychee#1774] [Q&A] [answered]
Hi
It is my second try to upgrade old version to latest 4 version but still unsuccessful. I dont know what or where id did went wrong or what i miss to do.

I read this documentation https://lycheeorg.github.io/docs/upgrade.html

in .env file i did make change
`# Table prefix (e.g. lychee_) of a Lychee v3 instance for migration
DB_OLD_LYCHEE_PREFIX=pics__lychee_`

in old conf there is
`$dbTablePrefix = 'pics_'; // Table prefix`

![mysql](https://user-images.githubusercontent.com/28824069/224494692-8f9b2bd8-dfd3-4cf3-8bcf-fe1eca7559d6.PNG)
![migrate error](https://user-images.githubusercontent.com/28824069/224494815-52e1829b-58a1-4647-a1ca-f59b0be11ad6.PNG)

So i dont understand what i should try more.
with best
Urmo

**Comment by qwerty287** [comment]
I don't really know about the v3 -> v4 migration, but can you try to just use `pics_` as prefix, also in the new config?

**Reply by bussekas** [reply]
I have tested many different perfix but still get same error and so thats why i make this post to understand what i did wrong.

**Comment by ildyria** [comment, marked as answer]
I just realized that it said nothing to Migrate. Which means that the migration has already been run.
Because we do not see those data, I am wondering if you are having a mix setup: e.g. current old install in one mysql DB and set up Lychee v4 to use sqlite as a new DB.

**Reply by bussekas** [reply]
i just did new install and connect to old db but seems to be that simple way is just create full new instance with new db and then move gallery to gallery from old to new instance and job is done. it just take some hard time to do it.
